### PR TITLE
Webpack update to version 4

### PR DESCRIPTION
**Pull request by katedev21**
Updating Webpack from v3 to v4 along with other necessary dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,31 +31,34 @@
...
     "standard": "^10.0.3",
-    "vue-loader": "^12.1.0",
+    "vue-loader": "^14.2.2",
     "vue-server-renderer": "^2.4.2",
     "vue-template-compiler": "^2.3.3",
     "vue-test-utils": "^1.0.0-beta.1",
-    "webpack": "^3.12.0",
-    "webpack-dev-server": "^2.11.5"
+    "webpack": "^4.46.0",
+    "webpack-cli": "^3.3.12",
```
Comment (pwolanin) [inline package.json:60]: do we need webpack cli?

File: package.json [hunk 2]
```diff
@@ -31,31 +31,34 @@
     "vuex": "^2.4.0"
   },
   "devDependencies": {
+    "autoprefixer": "^9.8.8",
```
Comment (pwolanin) [inline package.json:34]: This was not an explicit dependency before, but looks like it was used - does it need to be here?

**Comment by katedev21** [comment]
Npm 6 uses lock file version 1. By default, node 14 installs with npm 8.15.0. Npm 8 uses lock file version 2. By updating the Dockerfile to install npm 8.15.0, the npm version inside and outside of the container should match.

File: package.json (hunk 1 above)
Comment (pwolanin) [reply package.json:60]: short answer - yes, it's needed now for npm run-script build

**Review by pwolanin — APPROVED** [review]

**Pull request merged**